dev/release_files/LICENSE

```diff
@@ -201,28 +201,1238 @@
+  
+       2. Redistributions in binary form must reproduce the above copyright
+          notice, this list of conditions and the following disclaimer in the
+          documentation and/or other materials provided with the distribution.
+  
+       3. The names of its contributors may not be used to endorse or promote
+          products derived from this software without specific prior written
+          permission.
+  
+     THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+     "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+     LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+     A PARTICULAR PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+     CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+     EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+     PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+     PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+     LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+     NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+     SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+  ===============================================================================
+  
+  The class "org.apache.commons.math3.exception.util.LocalizedFormatsTest" is
+  an adapted version of "OrekitMessagesTest" test class for the Orekit library
+  The "org.apache.commons.math3.analysis.interpolation.HermiteInterpolator"
+  has been imported from the Orekit space flight dynamics library.
+  
+  Th Orekit library is described at:
+    https://www.orekit.org/forge/projects/orekit
+  The original files are distributed under the terms of the Apache 2 license
+  which is: Copyright 2010 CS Communication & Systèmes
+
+  ===============================================================================
+
+Apache Commons Math 3.3:
+
+  The initial code for shuffling an array (originally in class
+  "org.apache.commons.math3.random.RandomDataGenerator", now replaced by
+  a method in class "org.apache.commons.math3.util.MathArrays") was
+  inspired from the algorithm description provided in
+  "Algorithms", by Ian Craw and John Pulham (University of Aberdeen 1999).
+  The textbook (containing a proof that the shuffle is uniformly random) is
+  available here:
+  http://citeseerx.ist.psu.edu/viewdoc/download;?doi=10.1.1.173.1898&rep=rep1&type=pdf
+  ===============================================================================
+
+---
+
+Apache HBase (hbase-server)
+
+  This project bundles a copy of the JQuery minified javascript library version
+  1.8.3 under the terms of the MIT license.
+
+    Copyright 2012 jQuery Foundation and other contributors
+    http://jquery.com/
+
+  This project bundles a derivative image for our Orca Logo. This image is
+  available under the Creative Commons By Attribution 3.0 License.
```

Comment:
No, it wasn't. I think I missed the full license text here too. hbase-server.jar might also need looking at for the same reason (mental note).